### ISSUE TITLE
[bug]: jc parser cannot parse YAML field values that contain equal signs

### ISSUE THREAD

**Issue by lunrenyi** [kellyjonbrazil/jc#614]
- jc version: 1.25.3
- t.yml:
  ```yaml
  - Dec: 61
    Oct: "075"
    Hex: "0x3d"
    Ctrl: ""
    Acronym: =
    Description: =
    Description-zh: 等号
  ```
- Minimizing the case: `printf "%s\n" '- a: 1' '  b: =' | jc -p --yaml`
<img width="811" alt="image" src="https://github.com/user-attachments/assets/62c8ebf2-5a34-4b54-ada6-5db3d0287c01">

**Comment by kellyjonbrazil** [comment]
I have a fix for this in the `dev` branch. You can test by copying the `yaml.py` file to your plugin directory:

https://github.com/kellyjonbrazil/jc/blob/dev/jc/parsers/yaml.py

- Linux/unix: `$HOME/.local/share/jc/jcparsers`
- macOS: `$HOME/Library/Application Support/jc/jcparsers`
- Windows: `$LOCALAPPDATA\jc\jc\jcparsers`

Could you stress test this out? If you have any good YAML documents I can add them to my test suite. Thanks!

**Comment by kellyjonbrazil** [comment]
@georgettica @chriscroome could you test this latest YAML parser? I made a couple changes to support values that are only an equal sign ("="). I just want to make sure this change doesn't cause any other issues before rolling it out. If you have any good YAML files you would like me to add to the test suite, feel free to attach them. Of course, these will be public files, so if you do send any, please make sure they have no private data. Thanks!

**Comment by lunrenyi** [comment]
> I have a fix for this in the `dev` branch. You can test by copying the `yaml.py` file to your plugin directory:
> 
> https://github.com/kellyjonbrazil/jc/blob/dev/jc/parsers/yaml.py
> 
> * Linux/unix: `$HOME/.local/share/jc/jcparsers`
> * macOS: `$HOME/Library/Application Support/jc/jcparsers`
> * Windows: `$LOCALAPPDATA\jc\jc\jcparsers`
> 
> Could you stress test this out? If you have any good YAML documents I can add them to my test suite. Thanks!

Hey, thanks, it'll work!
<img width="566" alt="image" src="https://github.com/user-attachments/assets/84803a6c-fe4d-44b2-b53c-04b95ef7532f">

**Comment by kellyjonbrazil** [comment]
Now available in release v1.25.4
